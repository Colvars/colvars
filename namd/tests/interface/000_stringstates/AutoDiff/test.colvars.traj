# step         one                   fa_one                
           0    2.13236862431278e+00  0.00000000000000e+00 
           1    2.13245751463822e+00 -2.89496599179517e+01 
           2    2.13267749923821e+00 -3.31817108750264e+01 
           3    2.13306271104623e+00 -3.67376758031010e+01 
           4    2.13362553120190e+00 -3.96044965709237e+01 
           5    2.13435489235419e+00 -4.17550404800491e+01 
           6    2.13521727275308e+00 -4.31864384359511e+01 
           7    2.13615933633610e+00 -4.39214019832102e+01 
           8    2.13711157279848e+00 -4.40035034783994e+01 
           9    2.13799240954768e+00 -4.34912898646446e+01 
          10    2.13871247289520e+00 -4.24532563871765e+01 
          11    2.13917889857452e+00 -4.09645185510267e+01 
          12    2.13929973497631e+00 -3.91051042253309e+01 
          13    2.13898846493305e+00 -3.69590309268313e+01 
          14    2.13816852527797e+00 -3.46131556561760e+01 
          15    2.13677750976923e+00 -3.21550548008751e+01 
          16    2.13477057600232e+00 -2.96696321318850e+01 
          17    2.13212255549648e+00 -2.72348223860342e+01 
          18    2.12882840546139e+00 -2.49171916394546e+01 
          19    2.12490189619572e+00 -2.27683351080122e+01 
          20    2.12037277642510e+00 -2.08229577275481e+01 
# step         one                   fa_one                
          20    2.12037277642510e+00  0.00000000000000e+00 
          21    2.11544271621882e+00  1.55685198357011e+01 
          22    2.11053109496424e+00  1.49732643721063e+01 
          23    2.10564809928765e+00  1.43571942188257e+01 
          24    2.10079970227382e+00  1.37987285482642e+01 
          25    2.09598736281719e+00  1.33331173079539e+01 
          26    2.09120837138683e+00  1.29742647520364e+01 
          27    2.08645727541851e+00  1.27169689629116e+01 
          28    2.08172838289812e+00  1.25377732593621e+01 
          29    2.07701914056579e+00  1.23968049923061e+01 
          30    2.07233397071747e+00  1.22413809978373e+01 
          31    2.06768794093357e+00  1.20116466085310e+01 
          32    2.06310948666284e+00  1.16480621878565e+01 
          33    2.05864143959295e+00  1.10997205734801e+01 
          34    2.05433986583228e+00  1.03319813370927e+01 
          35    2.05027079223152e+00  9.33122313964807e+00 
          36    2.04650549387451e+00  8.10539509006982e+00 
          37    2.04311550279864e+00  6.68000010013434e+00 
          38    2.04016847949489e+00  5.09109929626650e+00 
          39    2.03772551919320e+00  3.37821361066347e+00 
          40    2.03583969528738e+00  1.57963802415380e+00 
# step         one                   fa_one                
          20    2.03583969528738e+00 -2.08229577275480e+01 
          21    2.03438314383169e+00 -2.16143395365368e+01 
          22    2.03323634173075e+00 -2.25233835035229e+01 
          23    2.03244195140922e+00 -2.35126646663931e+01 
          24    2.03202876525377e+00 -2.45381833000924e+01 
          25    2.03200708207517e+00 -2.55492788973770e+01 
          26    2.03236463476459e+00 -2.64910236683028e+01 
          27    2.03306411065418e+00 -2.73086633579748e+01 
          28    2.03404310547835e+00 -2.79532040785819e+01 
          29    2.03521691798102e+00 -2.83869421312746e+01 
          30    2.03648407605107e+00 -2.85878373517011e+01 
          31    2.03773389328150e+00 -2.85517208575191e+01 
          32    2.03885493484829e+00 -2.82918958325225e+01 
          33    2.03974309274381e+00 -2.78362553969507e+01 
          34    2.04030808975232e+00 -2.72225823716478e+01 
          35    2.04047757077894e+00 -2.64929683829132e+01 
          36    2.04019848919025e+00 -2.56884837850437e+01 
          37    2.03943606540642e+00 -2.48450516467789e+01 
          38    2.03817105681471e+00 -2.39911203725986e+01 
          39    2.03639641391554e+00 -2.31474142350967e+01 
          40    2.03411446660596e+00 -2.23285614961233e+01 
# step         one                   fa_one                
          40    2.03411446660596e+00 -2.23285614961230e+01 
          41    2.03133542211679e+00 -2.15457546742956e+01 
          42    2.02807738948757e+00 -2.08094244469945e+01 
          43    2.02436747919983e+00 -2.01309248821246e+01 
          44    2.02024294484702e+00 -1.95225111774162e+01 
          45    2.01575128265373e+00 -1.89957643054531e+01 
          46    2.01094849862541e+00 -1.85590777519031e+01 
          47    2.00589538841980e+00 -1.82151988203128e+01 
          48    2.00065242648937e+00 -1.79598284718619e+01 
          49    1.99527441604081e+00  0.00000000000000e+00 
          50    1.99007868248320e+00 -1.17687075716583e+01 
          51    1.98491160961488e+00 -1.26939462823049e+01 
          52    1.97977362495473e+00 -1.33199730082051e+01 
          53    1.97466058699500e+00 -1.37226439402165e+01 
          54    1.96956698057744e+00 -1.39735634464329e+01 
          55    1.96448934159629e+00 -1.41453863699019e+01 
          56    1.95942870824248e+00 -1.43022389898552e+01 
          57    1.95439148268945e+00 -1.44911545738228e+01 
          58    1.94938864077374e+00 -1.47378189697373e+01 
          59    1.94443365538142e+00 -1.50468046159132e+01 
          60    1.93953985541366e+00 -1.54059352497904e+01 
