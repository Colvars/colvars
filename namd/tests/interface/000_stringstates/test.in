colvarsTrajFrequency 1
colvarsRestartFrequency 10
indexFile index.ndx


colvar {

    name one

    outputAppliedForce on

    width 0.5
    lowerBoundary -10.0
    upperBoundary 10.0

    distanceZ {
        main {
            indexGroup group3
        }
        ref {
            indexGroup group4
        }
    }

} 


abf {
    colvars one
    fullSamples 1
}
