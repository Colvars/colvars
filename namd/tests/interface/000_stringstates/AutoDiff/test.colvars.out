colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: The restart output state file will be "test.tmp.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # indexFile = "index.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # sourceTclFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "one"
colvars:   Initializing a new "distanceZ" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "main".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group3"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "main".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "main" defined with 4 atoms requested: total mass = 54.028, total charge = -0.4.
colvars:       Initializing atom group "ref".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group4"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "ref".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "ref" defined with 4 atoms requested: total mass = 54.028, total charge = -0.4.
colvars:     # axis = ( 0 , 0 , 1 ) [default]
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.5
colvars:   # lowerBoundary = -10
colvars:   # upperBoundary = 10
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "abf" instance.
colvars:   # name = "abf1" [default]
colvars:   # colvars = { one }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   WARNING: ABF should not be run without a thermostat or at 0 Kelvin!
colvars:   # applyBias = on [default]
colvars:   # hideJacobian = off [default]
colvars:   Jacobian (geometric) forces will be included in reported free energy gradients.
colvars:   # fullSamples = 1
colvars:   # minSamples = 0 [default]
colvars:   # inputPrefix =  [default]
colvars:   # historyFreq = 0 [default]
colvars:   # shared = off [default]
colvars:   # updateBias = on [default]
colvars:   # maxForce =  [default]
colvars:   # integrate = on [default]
colvars:   Finished ABF setup.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 8 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - Colvars module:
colvars: - Colvars-NAMD interface:
colvars: - Optimal rotation via flexible fitting:
colvars: - distanceZ colvar component:
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - ABF colvar bias implementation:
colvars: - Internal-forces free energy estimator:
colvars:   Henin2010 https://doi.org/10.1021/ct9004432
colvars: 
colvars: - NAMD engine:
colvars: - Scalable center-of-mass computation (NAMD):
colvars:   Phillips2020 https://doi.org/10.1063/5.0014475
colvars: 
colvars: updating target temperature (T = 0 K).
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 8 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: updating target temperature (T = 0 K).
colvars: Current simulation parameters: initial step = 0, integration timestep = 1
colvars: Updating atomic parameters (masses, charges, etc).
colvars: Re-initialized atom group for variable "one":0/0. 4 atoms: total mass = 54.028, total charge = -0.4.
colvars: Re-initialized atom group for variable "one":0/1. 4 atoms: total mass = 54.028, total charge = -0.4.
colvars: The restart output state file will be "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
colvars: Saving state to output buffer.
colvars: Resetting the Collective Variables module.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # indexFile = "index.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # sourceTclFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "one"
colvars:   Initializing a new "distanceZ" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "main".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group3"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "main".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "main" defined with 4 atoms requested.
colvars:       Initializing atom group "ref".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group4"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "ref".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "ref" defined with 4 atoms requested.
colvars:     # axis = ( 0 , 0 , 1 ) [default]
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.5
colvars:   # lowerBoundary = -10
colvars:   # upperBoundary = 10
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "abf" instance.
colvars:   # name = "abf1" [default]
colvars:   # colvars = { one }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   WARNING: ABF should not be run without a thermostat or at 0 Kelvin!
colvars:   # applyBias = on [default]
colvars:   # hideJacobian = off [default]
colvars:   Jacobian (geometric) forces will be included in reported free energy gradients.
colvars:   # fullSamples = 1
colvars:   # minSamples = 0 [default]
colvars:   # inputPrefix =  [default]
colvars:   # historyFreq = 0 [default]
colvars:   # shared = off [default]
colvars:   # updateBias = on [default]
colvars:   # maxForce =  [default]
colvars:   # integrate = on [default]
colvars:   Finished ABF setup.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 8 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: updating target temperature (T = 0 K).
colvars: The restart output state file will be "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
colvars: Resetting the Collective Variables module.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # indexFile = "index.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # sourceTclFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "one"
colvars:   Initializing a new "distanceZ" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "main".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group3"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "main".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "main" defined with 4 atoms requested.
colvars:       Initializing atom group "ref".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group4"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "ref".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "ref" defined with 4 atoms requested.
colvars:     # axis = ( 0 , 0 , 1 ) [default]
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.5
colvars:   # lowerBoundary = -10
colvars:   # upperBoundary = 10
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "abf" instance.
colvars:   # name = "abf1" [default]
colvars:   # colvars = { one }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   WARNING: ABF should not be run without a thermostat or at 0 Kelvin!
colvars:   # applyBias = on [default]
colvars:   # hideJacobian = off [default]
colvars:   Jacobian (geometric) forces will be included in reported free energy gradients.
colvars:   # fullSamples = 1
colvars:   # minSamples = 0 [default]
colvars:   # inputPrefix =  [default]
colvars:   # historyFreq = 0 [default]
colvars:   # shared = off [default]
colvars:   # updateBias = on [default]
colvars:   # maxForce =  [default]
colvars:   # integrate = on [default]
colvars:   Finished ABF setup.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 8 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: updating target temperature (T = 0 K).
colvars: ----------------------------------------------------------------------
colvars: Loading state from formatted string.
colvars:   Restarting collective variable "one" from value: 2.12037
colvars:   Restarted abf bias "abf1" with step number 20.
colvars: ----------------------------------------------------------------------
colvars: The restart output state file will be "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
colvars: Resetting the Collective Variables module.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # indexFile = "index.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # sourceTclFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "one"
colvars:   Initializing a new "distanceZ" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "main".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group3"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "main".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "main" defined with 4 atoms requested.
colvars:       Initializing atom group "ref".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group4"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "ref".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "ref" defined with 4 atoms requested.
colvars:     # axis = ( 0 , 0 , 1 ) [default]
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.5
colvars:   # lowerBoundary = -10
colvars:   # upperBoundary = 10
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "abf" instance.
colvars:   # name = "abf1" [default]
colvars:   # colvars = { one }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   WARNING: ABF should not be run without a thermostat or at 0 Kelvin!
colvars:   # applyBias = on [default]
colvars:   # hideJacobian = off [default]
colvars:   Jacobian (geometric) forces will be included in reported free energy gradients.
colvars:   # fullSamples = 1
colvars:   # minSamples = 0 [default]
colvars:   # inputPrefix =  [default]
colvars:   # historyFreq = 0 [default]
colvars:   # shared = off [default]
colvars:   # updateBias = on [default]
colvars:   # maxForce =  [default]
colvars:   # integrate = on [default]
colvars:   Finished ABF setup.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 8 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: updating target temperature (T = 0 K).
colvars: Restarted abf bias "abf1" with step number 40.
colvars: The restart output state file will be "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
